### PR TITLE
Added "build-watch" command. Added ".git" and "dist" directories to .gitignore

### PR DESCRIPTION
**Pull request by tzoky07**
Added the .git and the dist/ directories to the .gitignore file
Added "build-watch" command based on watchify (https://github.com/substack/watchify), I've been using this for a while in my projects.
Removed dist/ directory from the cloned code. It can be built locally anyway so there is no use of it (eventually the dist/ folder can be created each time a new version is released)

### CODE REVIEW

**Comment by zenorocha** [comment]
Hey @tzoky07, thanks for your PR. 

There's no need to ignore `.git` and we can't remove `dist` because of Bower.

The only line that can be merged is this:

```
"build-watch": "watchify src/clipboard.js -o dist/clipboard.js -v"
```

Could you please amend your commit with only that?

**Comment by tzoky07** [comment]
Sure, I was just wondering. Would it be too much if I would include exorcist aswell ? It can build the source maps on the fly (so while the project size increases, debugging will still be easy).
It would be something like this

    watchify main.js -o 'exorcist dist/clipboard.js.map > dist/clipboard.js' -d -v

How does it sound to you ?

File: src/clipboard.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ class Clipboard extends Emitter {
         if (this.clipboardAction) {
             this.clipboardAction = null;
         }
-
+        // TODO: Find a way to reuse this object
```
Comment (zenorocha) [inline src/clipboard.js]: No need for this.

File: package.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
     "build": "npm run build-debug && npm run build-min",
     "build-debug": "browserify src/clipboard.js -s Clipboard -o dist/clipboard.js",
     "build-min": "uglify -s dist/clipboard.js -o dist/clipboard.min.js",
+    "build-watch": "echo 'Watching for file changes' && watchify src/clipboard.js -o dist/clipboard.js -v",
```
Comment (zenorocha) [inline package.json]: No need for this echo command too.

File: dist/clipboard.js [hunk 3]
```diff
@@ -574,5 +574,4 @@ var Clipboard = (function (_Emitter) {
 exports['default'] = Clipboard;
 module.exports = exports['default'];
 
-},{"./clipboard-action":6,"delegate-events":1,"tiny-emitter":5}]},{},[7])(7)
-});
+},{"./clipboard-action":6,"delegate-events":1,"tiny-emitter":5}]},{},[7]);
```
Comment (zenorocha) [inline dist/clipboard.js]: No need to recompile that file again.

**Comment by zenorocha** [comment]
I guess where are good without `exorcist` for now.

**Comment by tzoky07** [comment]
Alright, I've already committed the requested changes and rebuilt the project. 

Regards, 
Alex

**Comment by zenorocha** [comment]
Have you seen the comments I made?

**Comment by tzoky07** [comment]
Yep, I'll fix the commits as soon as I get home from work. Something like 1 hour or so.
P.S. : This is my first pull request so sorry if I screwed up anything

**Comment by tzoky07** [comment]
Fixed the commits. I think the PR's now ready

Cheers,
Alex

**Comment by zenorocha** [comment]
Perfect, that's very good for a first PR! Congratulations @tzoky07 and thanks for your help ;)

**Pull request merged**